sdk/wallet-standard/README.md

```diff
@@ -3,4 +3,4 @@
 A suite of standard utilities for implementing wallets and libraries based on the
 [Wallet Standard](https://github.com/wallet-standard/wallet-standard/).
 
-**Documentation:** https://sui-wallet-kit.vercel.app/advanced/wallet-standard
+**Documentation:** https://sui-wallet-kit.vercel.app/wallet-kit/advanced/wallet-standard
```

Comment:
I think the new domain for the docs is `https://sui-typescript-docs.vercel.app`.   cc @Jordan-Mysten